src/ProjectTemplates/README.md

```diff
@@ -10,3 +10,55 @@ To update project template JavaScript dependencies:
     ```
 
 To add a new dependency, run `npm install <package-name>` and update the `scripts` section in `package.json` to specify how the new dependency should be copied into its template.
+
+# Testing AI templates locally
+
+Regardless of how you run AI templates locally, you must first generate NuGet packages for `Microsoft.Extensions.AI` projects defined within this repo:
+```sh
+.\build.cmd -vs AI -noLaunch # Generate an SDK.sln for Microsoft.Extensions.AI* projects
+.\build.cmd -build -pack     # Build a NuGet package for each project
+```
+
+**Note:** Since package versions don't change between local builds, it may be necessary to occasionally delete `Microsoft.Extensions.AI*` packages from your local nuget cache, especially if you're making changes to these packages. An example of how to do this in PowerShell is:
+```pwsh
+Remove-Item ~\.nuget\packages\microsoft.extensions.ai* -Recurse -Force
+```
```

Comment:
It's unfortunate that this is the case, but I'm not aware of a better alternative. Maybe we could create scripts that build/run the template using a local nuget package cache. The cache would get cleared (or partially cleared) when the build detects that local packages have been updated. But I'm not sure if the effort to create and maintain that infrastructure is worth it, and I'm inclined to keep things simple for now.